CHANGELOG.md

```diff
@@ -31,9 +32,13 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#5779](https://github.com/thanos-io/thanos/pull/5779) Objstore: Support specifying S3 storage class.
 - [#5741](https://github.com/thanos-io/thanos/pull/5741) Query: add metrics on how much data is being selected by downstream Store APIs.
 - [#5673](https://github.com/thanos-io/thanos/pull/5673) Receive: Reload tenant limit configuration on file change.
+- [#5749](https://github.com/thanos-io/thanos/pull/5749) Query-Frontend: added small LRU cache to cache query analysis results.
 
 ### Changed
 
+- [#5738](https://github.com/thanos-io/thanos/pull/5738) Global: replace crypto/sha256 with minio/sha256-simd to make it faster.
```

Comment:
To make what faster? How? :smile: :racing_car: 
```suggestion
- [#5738](https://github.com/thanos-io/thanos/pull/5738) Global: Replace `crypto/sha256` with `minio/sha256-simd` to make it faster.
```

Reply:
It enables parallel computation via SIMD. I guess we can mention that

Reply:
Removed `it` here and wrote specifically that this makes hash calculation faster in reloader and metadata packages.